commands/Noop.java

```diff
@@ -1,27 +1,23 @@
 package org.usfirst.frc4904.standard.commands;
 
+import java.util.Set;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.Command;
 
-import edu.wpi.first.wpilibj.command.Command;
+public class Noop implements Command {
+	public void initialize() {}
 
-public class Noop extends Command {
-	public Noop() {
-		super("Noop");
-	}
-
-	@Override
-	protected void initialize() {}
-
-	@Override
-	protected void execute() {}
+	public void execute() {}
 
-	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		return true;
 	}
 
-	@Override
-	protected void end() {}
+	public void end() {}
```

Comment:
please remove `interrupted()`, add `@Override`, and add a `boolean` argument to `end()`. End now takes a `boolean` which specifies if the command ended by interruption

Reply:
just delete end